package-lock.json

```diff
@@ -12,9 +12,9 @@
         "packages/components"
       ],
       "dependencies": {
-        "@floating-ui/react-dom": "0.7.1",
+        "@floating-ui/react-dom": "0.7.2",
```

Comment:
Would you mind reverting the changes in this file? I've submitted a separate PR to address them

https://github.com/mattermost/mattermost-webapp/pull/10832

Reply:
Sorry my mistake. I wasnt planning to commit these changes. I mistakenly committed this